proposals/new-project-addition-process.md

```diff
@@ -0,0 +1,36 @@
+## Motivation
```

Comment:
I think, we also require to establish process for Project Onboarding if Kubeflow community accepted this project.
For example, for Spark Operator we had this tracking issue: https://github.com/kubeflow/spark-operator/issues/1928